plasmapy/formulary/lengths.py

```diff
@@ -9,7 +9,7 @@
 from astropy.constants.si import c, e, eps0, k_B
 
 from plasmapy import particles
-from plasmapy.formulary.frequencies import gyrofrequency, plasma_frequency
+from plasmapy.formulary import frequencies, speeds
```

Comment:
For these "core" formulary modules I'm purposefully import the module and not just it's contents to avoid circular imports.  This specifically impacts the `lengths` and `speeds` modules since they both need functionality from each other.